### DISCUSSION TITLE
Opacity tokens not working properly/no autocompletion

### DISCUSSION THREAD

**Discussion by Msordet** [chakra-ui/panda#1138] [Q&A] [answered]
Hi,

I'm trying to use custom tokens for opacity and there is no autocompletion or generated css it seems.

Reproduction: https://play.panda-css.com/UkVbNBdDwc

Is my understanding of opacity tokens wrong?

**Comment by anubra266** [comment, marked as answer]
@Msordet The token is generated correctly
![CleanShot 2023-07-31 at 20 31 49@2x](https://github.com/chakra-ui/panda/assets/30869823/dc3feb9a-5392-4800-89ec-60c55eecba4b)

But style does not apply it
![CleanShot 2023-07-31 at 20 32 27@2x](https://github.com/chakra-ui/panda/assets/30869823/5efeeb7a-aab4-4dec-a28b-715e827bf54b)

Please open an issue

**Reply by anubra266** [reply]
![CleanShot 2023-07-31 at 20 34 21@2x](https://github.com/chakra-ui/panda/assets/30869823/fb1d67e7-a15a-4924-bb6e-3f5721ec971f)
Found the issue. opacity is not defined in the default preset utilities

**Reply by anubra266** [reply]
@Msordet I've opened a PR https://github.com/chakra-ui/panda/pull/1144
